content/client-lib-development-guide/features.textile

```diff
@@ -1901,6 +1914,13 @@ class HttpPaginatedResponse // RSC19b
   errorCode: Int // HP6
   errorMessage: String // HP7
   headers: Dict<String, String> // HP8
+  
+class PluggableCodec // PC2
+  // Empty class/interface. Pluggable codecs are not expected to share any common interface.
+  // Still it is better to have a common base type for pluggable codecs in statically typed languages.
+  
+class VCDiffDecoder
+  decode([byte] delta, [byte] base) -> [byte]
```

Comment:
Pending agreement as to whether these args should be reversed.

see https://github.com/ably/ably-java/pull/532#discussion_r350663132

Reply:
My preference remains to reverse these args but it feels like I'm alone in this. :cry: